dawn/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ if(EXISTS "${LOC_PATH}")
 endif()
 
 # Path for external CMake files
-list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake ${CMAKE_CURRENT_SOURCE_DIR}/../cmake)
```

Comment:
My only concern here is that this breaks the isolation of dawn. Since we say gtclang -> dawn -> LLVM/Clang shouldn't `FindLLVM.cmake` go in `dawn/cmake/`?

Reply:
The problem with that is that gtclang still needs the `FindLLVM.cmake` and `FindClang.cmake` files because it installs them in the build folder.
See here: https://github.com/MeteoSwiss-APN/dawn/blob/c7a92a44ff14b851ca0f39d95bb3660f9657b7dd/gtclang/CMakeLists.txt#L183 and the following 3 lines.

So gtclang needs access to these FindCmake files, OR we make sure that dawn installs these FindCmake files into the right place.

Reply:
hmm i wonder why we install those in the first place ... feels like we could remove that install